client/profile-wizard/index.js

```diff
@@ -120,7 +120,7 @@ class ProfileWizard extends Component {
 				profileItems.product_types !== null,
 		} );
 		steps.push( {
-			key: 'business-features',
+			key: 'business-details',
```

Comment:
I updated this and now the `business-features` tracks are handled in the `selective-bundle`. This does change the param name in the url to `business-details` as well.
I don't think this is an issue, as I am not aware of any links that open a specific step in the onboarding wizard, but just mentioning that here incase someone else does.

Reply:
I don't think think this will cause any major issues.  If it were a more commonly visited page, we might be breaking some bookmarks but given this page is on average seen only ~1 time per site, I think we should be fine changing this.